test/e2e/specs/editor/blocks/links.spec.js

```diff
@@ -58,7 +58,72 @@ test.describe( 'Links', () => {
 		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
 	} );
 
-	test.describe( 'should contain a label when it should open in a new tab', () => {} );
+	test( 'should contain a label when it should open in a new tab', async ( {
+		page,
+		editor,
+		pageUtils,
+		linkControl,
+	} ) => {
+		await linkControl.clickBlockAppender();
+		await page.keyboard.type( 'This is WordPress' );
+		// Select "WordPress".
+		await pageUtils.pressKeys( 'shiftAlt+ArrowLeft' );
+		await pageUtils.pressKeys( 'primary+k' );
+		await page.keyboard.type( 'w.org' );
+
+		// Link settings open
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Space' );
+
+		// Navigate to and toggle the "Open in new tab" checkbox.
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Space' );
+
+		// Confirm that focus was not prematurely returned to the paragraph on
+		// a changing value of the setting.
+		await expect( page.getByLabel( 'Open in new tab' ) ).toBeFocused();
+
+		// Submit link. Expect that "Open in new tab" would have been applied
+		// immediately.
```

Comment:
```suggestion
		// Submit link. Expect that "Open in new tab" setting is persisted.
```

I think the "immediately" was because previously toggling the setting would immediately submit the control. This no longer happens.